app/models/loan_response.rb

```diff
@@ -54,9 +54,10 @@ def field_attributes
     end
   end
 
+  # Checks if response is blank, including any descenants if this is a group.
   def blank?
     if group?
-      loan_question.descendants.all? { |i| loan_response_set.response(i.id).blank? }
+      question.children.all?(&:blank?)
```

Comment:
Using `question.children.all?(&:blank?)` causes a regression. The questions show up in the loan memo. If a question or group is unanswered, it should not appear in the loan memo. This code change makes these empty groups appear in the printed loan memo, which should not be the case. They show up both in the table of contents and in the following pages, but should not appear at all.

Because `descendants` is a banned method, this would need to be revised in order for this to work with the permitted methods while not causing a regression.

Reply:
I was unable to make a fix for this regression.